scripts/enum/item.lua

```diff
@@ -248,9 +249,11 @@ xi.item =
     DARKSTEEL_INGOT                     = 654,
     ADAMAN_INGOT                        = 655,
     BEASTCOIN                           = 656,
+    TAMA_HAGANE                         = 657,
```

Comment:
Please use the long names used in the database. Example here:

LUMP_OF_TAMA_HAGANE

Reply:
Figures I miss one, 837 lines changed, so far only one. Do let me know if there are others.

Addressed
https://github.com/LandSandBoat/server/compare/7834d0dabb272fee24f052abf6672e8dc853f8a3..febeca5d1179fc3f2391c217fc331d337148d2bb